index.js

```diff
@@ -1,525 +1,559 @@
+ * @property {string|RichEmbed|MessageEmbed} [banMessage='**{user_tag}** has been banned for spamming.'] Message that will be sent in chat upon banning a user.
+ * @property {string|RichEmbed|MessageEmbed} [muteMessage='**{user_tag}** has been muted for spamming.'] Message that will be sent in chat upon muting a user.
+ *
+ * @property {boolean} [errorMessages=true] Whether the error messages, when the bot doesn't have enough permissions, must be sent or not
+ * @property {string|RichEmbed|MessageEmbed} [kickErrorMessage='Could not kick **{user_tag}** because of improper permissions.'] Message that will be sent in chat when the bot doesn't have enough permissions to kick the member.
+ * @property {string|RichEmbed|MessageEmbed} [banErrorMessage='Could not ban **{user_tag}** because of improper permissions.'] Message that will be sent in chat when the bot doesn't have enough permissions to ban the member.
+ * @property {string|RichEmbed|MessageEmbed} [muteErrorMessage='Could not mute **{user_tag}** because of improper permissions or the mute role couldn't be found.'] Message that will be sent in chat when the bot doesn't have enough permissions to mute the member.
+ *
+ * @property {number} [maxDuplicatesWarning=7] Amount of duplicate messages that trigger a warning.
+ * @property {number} [maxDuplicatesKick=11] Amount of duplicate messages that trigger a kick.
+ * @property {number} [maxDuplicatesMute=9] Amount of duplicate messages that trigger a kick.
+ * @property {number} [maxDuplicatesBan=12] Amount of duplicate messages that trigger a ban.
+ *
+ * @property {number} [deleteMessagesAfterBanForPastDays=1] Amount of days in which old messages will be deleted. (1-7)
+ *
+ * @property {Array<string>} [exemptPermissions=[]] Bypass users with at least one of these permissions
+ * @property {boolean} [ignoreBots=true] Whether bot messages are ignored
+ * @property {boolean} [verbose=false] Extended Logs from module (recommended)
+ * @property {boolean} [debug=false] Whether to run the module in debug mode
+ *
+ * @property {Array<string>|function} [ignoredUsers=[]] Array of string user IDs that are ignored
+ * @property {Array<string>|function} [ignoredRoles=[]] Array of string role IDs or role name that are ignored
+ * @property {Array<string>|function} [ignoredGuilds=[]] Array of string Guild IDs that are ignored
+ * @property {Array<string>|function} [ignoredChannels=[]] Array of string channels IDs that are ignored
+ *
+ * @property {boolean} [warnEnabled=true] If false, the bot won't warn users
+ * @property {boolean} [kickEnabled=true] If false, the bot won't kick users
+ * @property {boolean} [banEnabled=true] If false, the bot won't ban users
+ * @property {boolean} [muteEnabled=true] If false, the bot won't mute users
+ * 
+ * @property {boolean} [modlog=false] If false, the bot won't send moderation logs in a specefic channel.

```

Comment:
Here it's `modlog` and line 95 it's `modlogEnabled`. I think the best solution is to set `modLogChannel` to `null` by default. And if it has a value, we can consider moderation logs enabled.